### ISSUE TITLE
Why not using bs-material-ui?

### ISSUE THREAD

**Issue by dmtrKovalenko** [Tevinthuku/material-ui-reason-react-demo#1]
It looks like https://github.com/jsiebern/bs-material-ui provide working typings for material-ui. It will be cool to add them to the example

**Comment by Tevinthuku** [comment]
Yes, I agree this library has better binding support for all Material-UI components. 
I do mention this in the readme of this repo https://github.com/Tevinthuku/material-ui-reason-react-demo/blob/master/README.md.

The aim of this demo was to set up a minimal project with material-ui and reason-ml without using any 
 other external libraries.
